### PR TITLE
doc: add etcd to test dependencies

### PR DESCRIPTION
**Pull request by patapenka-alexey**
etcd and lua have been added to the dependencies list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,6 +178,8 @@ To run tests:
 -   [docker](https://docs.docker.com/engine/install/)
 -   [NodeJS](https://nodejs.org/en/download)
 -   [CMake](https://cmake.org/install/)
+-   [etcd](https://etcd.io/docs/v3.5/install/)
+-   [lua](https://www.lua.org/download.html)
```
Comment (oleg-jukovec) [inline README.md:182]: Why? Lua shouldn't be required.

**Comment by oleg-jukovec** [comment]
Please, rebase to a latest commit of the `master` branch.

File: README.md (hunk 1 above)
Comment (patapenka-alexey) [reply README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -178,6 +178,7 @@ To run tests:
 -   [docker](https://docs.docker.com/engine/install/)
 -   [NodeJS](https://nodejs.org/en/download)
 -   [CMake](https://cmake.org/install/)
+-   [etcd](https://etcd.io/docs/v3.5/install/)
```
Comment (oleg-jukovec) [inline README.md:181]: The tests won't work with etcd v2, we need to highlight it.
```suggestion
-   [etcd 3+](https://etcd.io/docs/v3.5/install/)
```

**Review by oleg-jukovec — APPROVED** [review]

**Review by DerekBum — APPROVED** [review]

**Pull request merged**